src/scss/overview.scss

```diff
@@ -8,10 +8,6 @@
   color: #e32727;
 }
 
-.rsTl-topic-section {
-  margin-bottom: 35px;
```

Comment:
Can we not just change this to `padding-bottom: 35px` and then we don't need to modify zoolander at all? It seemed to work when I tested in my browser.

Reply:
well, that's just another means of skinning a cat and if you want to go all making sense like that. Yes. I just started off trying to resolve this from zoolander first and then realized .rsTl-topic-section existed in solve needed to have changes. it just didn't occur to me to attempt to consolidate or change the mindset I already had. I'll give that a try now.